### PR TITLE
ARR-4426 Update REST Plugin to 4.0.1

### PR DESCRIPTION
**Pull request by clowe-r7**
## Requirements

Workflow contributors, verify you have completed the following items by checking them off:

### Checklist
- [ ] Workflow verified as working in a test
- [ ] Workflow bundle includes `Name_of_Workflow.icon` file, `help.md` file, `workflow.spec.yaml` file, and `screenshots` directory
- [ ] Workflow title should follow general [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Workflow includes a detailed Description
- [ ] Workflow step names follow [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Triggers, Decisions, and other workflow steps include descriptions where needed
- [ ] Workflow includes at least one `Final Report` artifact that summarizes workflow execution details
- [ ] Workflow bundle (directory) is named after the title of the workflow in snake case with first letters capitalized, e.g. `workflows/Name_of_Workflow/`
- [ ] `help.md` file includes Description, Key Features, Requirements, Setup, Technical Details, Version History, and References
- [ ] `workflow.spec.yaml` file includes `products`, `name`, `title`, `description`, `use_cases` tags, `keywords` tags, `source_url`, and the `name` for each screenshot matches a PNG image in the `screenshots` directory

### Assessment

Post your results with screenshots in this PR using the following style:

```
### Screenshots

Screenshots help your reviewers provide valuable feedback on your contribution. To add a screenshot here, simply drag and drop the PNG files from a file explorer.

#### Workflow Builder

<Insert Workflow Image Here>

#### Job

<Insert Job Image Here>

#### Artifact

<Insert Artifact Image Here>
```

### CODE REVIEW

File: workflows/MDR_Active_Response_Disable_User_with_Active_Directory/help.md [hunk 1]
```diff
@@ -37,8 +36,9 @@ _There is no troubleshooting information at this time_
 
 # Version History
 
-* 1.0.0 - Initial workflow
+* 1.0.2 - Upgrade HTTP Rest Plugin
```
Comment (mrinehart-r7) [inline workflows/MDR_Active_Response_Disable_User_with_Active_Directory/help.md:39]:
```suggestion
* 1.0.2 - Upgrade HTTP Requests and Active Directory LDAP plugins
```

File: workflows/MDR_Active_Response_Enable_User_with_Active_Directory/help.md [hunk 2]
```diff
@@ -33,8 +33,9 @@ _There is no troubleshooting information at this time_
 
 # Version History
 
-* 1.0.0 - Initial workflow
+* 1.0.2 - Update AD LDAP Plugin to 5.0.1
```
Comment (mrinehart-r7) [inline workflows/MDR_Active_Response_Enable_User_with_Active_Directory/help.md:36]:
```suggestion
* 1.0.2 - Update Active Directory LDAP Plugin to 5.0.1
```

File: workflows/MDR_Active_Response_Isolate_Sensor_with_VMware_Carbon_Black_EDR/help.md [hunk 3]
```diff
@@ -42,6 +41,7 @@ _There is no troubleshooting information at this time_
 
 # Version History
 
+* 1.0.1 - Update HTTP Requests Plugin
```
Comment (mrinehart-r7) [inline workflows/MDR_Active_Response_Isolate_Sensor_with_VMware_Carbon_Black_EDR/help.md:44]:
```suggestion
* 1.0.1 - Update HTTP Requests plugin
```

File: workflows/MDR_Active_Response_Router/MDR_Active_Response_Router.icon [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
...ck, and SMS.\n* Accelerate or cancel containment actions from your mobile or desktop devices via Slack.\n\n\n# Requirements\n\n* Insight Platform Organization-level API key\n* Python3 Script \n* Type Converter\n* REST\n* Active Directory LDAP\n* Microsoft Office 365 Email \n* Twilio (Optional) \n* Slack (ChatOps)\n\n\n# Documentation\n\n## Setup\n\nImport the workflow from the Rapid7 Extension Library and proceed through the Import Workflow wizard in InsightConnect. Import plugins, create or select connections, and rename the workflow as a part of the Import Workflow wizard as necessary.\n \n## Technical Details\n\nFor Technical Details, please refer to the import MDR Active Response Asset Quarantine Workflow page.\n\n## Troubleshooting\n\n|Plugin|Version|Count|\n|----|----|--------|\n|REST|4.1.0|5|\n|Python 3 Script|2.0.2|4|\n|Twilio|1.0.2|1|\n|Type Converter|1.5.1|1|\n|Microsoft Office 365 Email|4.1.4|1|\n|String Operations|1.3.0|2|\n\n_There is no troubleshooting information at this time_\n\n# Version History\n\n* 1.1.0 - Added Parameters and Updated HTTP Rest Plugin\n* 1.0.1 - Fix null issue affecting the filter step\n* 1.0.0 - Initial workflow\n\n\n# Links\n\n## References\n",
```
Comment (mrinehart-r7) [inline workflows/MDR_Active_Response_Router/MDR_Active_Response_Router.icon:11]:
```suggestion
        "description": "# Description\n\nThis workflow is part of the MDR Active Response add-on. When the MDR team identifies a threat, such as a compromised user, this workflow enables them to send an email, SMS, and Slack notification that you can review and respond to within a grace period of 10 minutes. This workflow also triggers the MDR Containment Disable User with Active Directory and MDR Containment Isolate Sensor in Cb Response workflows.\n\n# Key Features\n\n* Quarantine Asset with Insight Agent or Cb Response\n* Send real-time updates to actions using your preferred communication methods: phone, email, Slack, and SMS.\n* Accelerate or cancel containment actions from your mobile or desktop devices via Slack.\n\n\n# Requirements\n\n* Insight Platform Organization-level API key\n* Python3 Script \n* Type Converter\n* REST\n* Active Directory LDAP\n* Microsoft Office 365 Email \n* Twilio (Optional) \n* Slack (ChatOps)\n\n\n# Documentation\n\n## Setup\n\nImport the workflow from the Rapid7 Extension Library and proceed through the Import Workflow wizard in InsightConnect. Import plugins, create or select connections, and rename the workflow as a part of the Import Workflow wizard as necessary.\n \n## Technical Details\n\nFor Technical Details, please refer to the import MDR Active Response Asset Quarantine Workflow page.\n\n## Troubleshooting\n\n|Plugin|Version|Count|\n|----|----|--------|\n|REST|4.1.0|5|\n|Python 3 Script|2.0.2|4|\n|Twilio|1.0.2|1|\n|Type Converter|1.5.1|1|\n|Microsoft Office 365 Email|4.1.4|1|\n|String Operations|1.3.0|2|\n\n_There is no troubleshooting information at this time_\n\n# Version History\n\n* 1.1.0 - Added Parameters and Updated HTTP Rest Plugin\n* 1.0.1 - Fix null issue affecting the filter step\n* 1.0.0 - Initial workflow\n\n\n# Links\n\n## References\n",
```

Comment (mrinehart-r7) [reply workflows/MDR_Active_Response_Router/MDR_Active_Response_Router.icon:11]: Fixed a capital I in there

File: workflows/MDR_Active_Response_Router/help.md [hunk 5]
```diff
@@ -35,19 +35,19 @@ For Technical Details, please refer to the import MDR Active Response Asset Quar
...
-|Type Converter|1.5.1|1|
 |Microsoft Office 365 Email|4.1.4|1|
 |String Operations|1.3.0|2|
 
 _There is no troubleshooting information at this time_
 
 # Version History
 
-* 1.0.0 - Initial workflow
+* 1.0.2 - Update REST Plugin to 4.0.1
```
Comment (mrinehart-r7) [inline workflows/MDR_Active_Response_Router/help.md:48]:
```suggestion
* 1.0.2 - Update HTTP Requests plugin to 4.0.1
```

**Review by mrinehart-r7 — CHANGES_REQUESTED** [review]

**Review by mrinehart-r7 — APPROVED** [review]

**Review by bszczytko-r7 — APPROVED** [review]

**Pull request merged**